editor/project_manager.cpp

```diff
@@ -1518,6 +1583,11 @@ ProjectManager::ProjectManager() {
 			rename_btn->connect(SceneStringName(pressed), callable_mp(this, &ProjectManager::_rename_project));
 			project_list_sidebar->add_child(rename_btn);
 
+			duplicate_btn = memnew(Button);
+			duplicate_btn->set_text(TTR("Duplicate"));
+			duplicate_btn->connect(SceneStringName(pressed), callable_mp(this, &ProjectManager::_duplicate_project));
```

Comment:
You can do
```suggestion
			duplicate_btn->connect(SceneStringName(pressed), callable_mp(this, &ProjectManager::_duplicate_project_with_action).bind(POST_DUPLICATE_ACTION_NONE));
```
and remove `duplicate_project()` (or replace it).